src/api/index.js

```diff
@@ -2,6 +2,10 @@
 
 import axios from 'axios'
 
-export default {
+export let getNoteList = ( callBack ) => {
```

Comment:
Is it acceptable to have other APIs defined here as well?

Reply:
Yes, we can define more functions and export it.

Reply:
I understand. Thank you.